docs/install_on_kubernetes.md

```diff
@@ -0,0 +1,204 @@
+# Install the common service operator on vanila Kubernetes
+
+In this document, we will show you how to deploy and use the meta operator on the vanila Kubernetes.
```

Comment:
one minor comment, please use `doctoc` generate a ToC. We can fix this in another PR.